client/components/Header.jsx

```diff
@@ -110,8 +110,8 @@ const Header = () => {
         <NavLink to="/faqs" className={classes.link} activeStyle={activeStyle}>
           <Button className={classes.button}>FAQ</Button>
         </NavLink>
-        <NavLink to="/blog" className={classes.link} activeStyle={activeStyle}>
```

Comment:
Now that I'm thinking about it, we might want to keep the blog. So please just add a new NavLink for About, and keep the existing NavLink for Blog.

Reply:
Will do.